src/components/MovieDetail/MovieDetail.jsx

```diff
@@ -1,11 +1,17 @@
+import CharacterDetail from '../CharacterDetail'
+
 export default function MovieDetail({ movie }) {
     return (
         <div>
             <h1>{movie.title}</h1>
             <img src={movie.img} alt={movie.title}/>
             <p>{movie.description}</p>
-
-    {movie.characters.map(character => <h3>{character.name}</h3>)}
+        
```

Comment:
Agregar un título para saber que son los personajes al igual que arriba explicar que es la sinopsis
```suggestion
        <h3>Sinopsis</h3>
        <p>{movie.description}</p>
```